packages/ui/package.json

```diff
@@ -54,8 +54,8 @@
     "metro-config": "0.80.5",
     "metro-resolver": "0.80.5",
     "prism-react-renderer": "2.3.1",
-    "react": "18.3.1",
-    "react-dom": "18.3.1",
+    "react": "18.2.0",
```

Comment:
There are some issues when installing in the extension now, I think related to a conflicting react version:
![Screenshot 2024-06-04 at 09 51 49](https://github.com/leather-wallet/mono/assets/2938440/253bb1f4-e4b9-4d38-ba22-cbcde8fafec0)

I tried updating it here but it had a knock-on effect causing type issues in the native components.

I'll work on that in another branch and leave this open as a reference